lib/missing_validators/matchers/ensure_valid_ip_address_format_of.rb

```diff
@@ -0,0 +1,20 @@
+RSpec::Matchers.define :ensure_valid_ip_address_format_of do |attribute|
+  root = 'missing_validators.matchers.ensure_valid_ip_address_format_of'
+
+  match do |model|
+    model.send("#{attribute}=", 'invalid.ip.address')
+    model.valid?
+
+    if model.errors.to_hash.key?(attribute)
+      model.errors[attribute].include?(I18n.t('errors.messages.ip_address'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.